resources/UTXOracle.py

```diff
@@ -0,0 +1,730 @@
+
```

Comment:
Adding shebang will allow to launch it by `UTXOracle.py` instead of `python3 UTXOracle.py`.
```suggestion
#!/usr/bin/env python3
```